data/_ui/interfaces.txt

```diff
@@ -777,10 +777,8 @@ interface "hud"
 		from -70 320
 		color "medium"
 		align right
-
-
-
-	# Targets.
+	
+	# Radar
```

Comment:
```suggestion



	# RADAR
```

I generally prefer to keep 3 lines between different or unrelated sections. In this case, the stuff around the radar and the stuff around the flagship. And the slightly picky correction of making the radar all uppercase.... :)